users/vieting/jobs/fairseq_finetuny.py

```diff
@@ -0,0 +1,185 @@
+import soundfile
+import glob
+import os
+import random
+import subprocess
+import os
+
+from i6_core.lib import corpus
+
+from sisyphus import Job, Task, tk
+
+
+class CreateTaskDataJob(Job):
```

Comment:
```suggestion
class CreateFairseqLabeledDataJob(Job):
```
I think just task data is too generic. If you have a better suggestion than mine, feel free to propose it.